docs-conceptual/azps-4.6.0/release-notes-azureps.md

```diff
@@ -49,7 +49,7 @@ ms.custom: devx-track-azurepowershell
 * Added 'Get-AzDeploymentManagementGroupWhatIfResult' for getting ARM template What-If results at management Group scope
 * Added 'Get-AzTenantWhatIfResult' new cmdlet for getting ARM template What-If results at tenant scope
 * Overrode '-WhatIf' and '-Confirm' for 'New-AzManagementGroupDeployment' and 'New-AzTenantDeployment' to use ARM template What-If results
-* Fixed the behaviors of '-WhatIf' and '-Confirm' for new deployment cmdlets so they comply with False and 
+* Fixed the behaviors of '-WhatIf' and '-Confirm' for new deployment cmdlets so they comply with False and (missing text here)
```

Comment:
This seems to be a bug in our release note generator.
The correct sentence should be 
"Fixed the behaviors of -WhatIf and -Confirm for new deployment cmdlets so they comply with $WhatIfPreference and $ConfrimPreference"